### PR TITLE
Fixes #4

### LINKED ISSUE #4: Breaks when first argument is blank/null
Looks like the package breaks when `srcDir` is specified in the options but the first argument is blank/null. Example:

```
mix.react(null, {
    output: "path/to/bundle/",
    srcDir: "path/to/jsx",
    harmony: true
});
```

There's no error or warning, but this task won't run. The problem is this line:

```
src = "./" + utilities.buildGulpSrc(src, options.srcDir);
```

Returns this:

```
/path/to/jsx/undefined
```

I suggest changing the line to

```
src = "./" + utilities.buildGulpSrc(src, options.srcDir, "");
```

So that the `src` becomes:

```
/path/to/jsx/
```

### PR DESCRIPTION
**Pull request by charmeleon-zz**
Fix source directory path when srcDir is specified and the first argument is blank

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ elixir.extend('react', function (src, options) {
         output: config.jsOutput
     }, options);
 
-    src = "./" + utilities.buildGulpSrc(src, options.srcDir);
+    src = "./" + utilities.buildGulpSrc(src, options.srcDir, "");
```
Comment (joecohens) [inline index.js]: Hey! I think it'll be better to do `'**/*.+(js|jsx)'` instead

Comment (joecohens) [inline index.js]: Use single quotes please.

**Comment by joecohens** [comment]
PING @charmeleon

**Comment by charmeleon-zz** [comment]
Hi again @joecohens. I made changes you suggested and adjusted the PR

File: index.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ var gulp = require('gulp'),
 
 elixir.extend('react', function (src, options) {
     var config = this;
+    var filePattern = '/**/*.+(js|jsx)';
```
Comment (joecohens) [inline index.js:12]: The backslash at the beginning won't work on the search path

Comment (joecohens) [inline index.js:12]: I'll change it

**Comment by joecohens** [comment]
Thanks :)

**Pull request merged**